source/tutorial/kerberos.txt

```diff
@@ -225,19 +251,19 @@ Kerberos authentication on either Windows or macOS.
                      displayName: "MongoDB BI Connector"
 
             - Open :guilabel:`Services`.
-            - Set MongoDB to 
-              :guilabel:`Log on as: This account:` 
-              ``mongodb@EXAMPLE.COM`` 
-            - Set |bi| to 
-              :guilabel:`Log on as: This account:` 
+            - Set MongoDB to
+              :guilabel:`Log on as: This account:`
+              ``mongodb@EXAMPLE.COM``
```

Comment:
Not sure if this is wrong but the double-colon formatting here seems strange. Perhaps it makes more sense in context.

Reply:
It's in the UI that way.